app/src/main/res/drawable/ic_tutorial_img_2.xml

```diff
@@ -0,0 +1,11 @@
+<vector android:height="169dp" android:viewportHeight="168.55"
+    android:viewportWidth="238.08" android:width="238dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#878787" android:pathData="M83.69,76.54l-3.38,6c-1.9,3.38 -4.69,4.65 -8.52,4.08 -7,-1 -13.91,-2 -20.86,-3a10.51,10.51 0,0 1,-3.33 -1.1c0.29,-0.17 0.58,-0.36 0.88,-0.51 2.66,-1.29 5.31,-2.59 8,-3.87 4,-1.94 4.85,-3.57 4.09,-8.08l3.07,0.43a39.8,39.8 0,0 1,4.38 0.62c1.27,0.33 1.83,-0.09 2.42,-1.17 3.28,-5.95 6.66,-11.84 10,-17.78a10.86,10.86 0,0 1,7.75 -5.64c4.85,-1 9.66,-2.11 14.5,-3.15a11.82,11.82 0,0 1,8.25 1.18c8.21,4.47 16.32,9.12 24.47,13.71a7,7 0,0 1,3.07 3.57c3.75,9.21 7.51,18.4 11.19,27.63A7.39,7.39 0,0 1,135.88 95c-3.31,-8 -6.52,-16 -9.82,-24a3.8,3.8 0,0 0,-1.41 -1.64c-1.9,-1.17 -3.88,-2.22 -5.83,-3.32 -0.34,-0.18 -0.69,-0.34 -1.26,-0.62 0.71,3.23 1.35,6.21 2,9.2 2.08,9.51 4.26,19 6.19,28.53a80.82,80.82 0,0 1,1.12 10.23c0.27,3.78 0.34,7.58 0.55,11.38a2.57,2.57 0,0 0,0.62 1.35q9.55,11.4 19.16,22.76c1.95,2.3 2.85,4.79 1.9,7.75a7.34,7.34 0,0 1,-12.47 2.68c-2.53,-2.78 -4.88,-5.72 -7.31,-8.59 -4.43,-5.26 -8.74,-10.63 -13.33,-15.75a13.48,13.48 0,0 1,-3.42 -8.8c-0.18,-3.47 -0.61,-6.91 -1,-10.58l-4.49,0.88c-0.7,0.14 -1.4,0.36 -2.11,0.43a9.47,9.47 0,0 0,-6.11 3.27c-4.56,4.77 -9.28,9.39 -14,14A4.56,4.56 0,0 0,83.5 137c-1,8 -2,16.09 -2.92,24.14 -0.47,4 -3,6.74 -6.79,7.23a7.19,7.19 0,0 1,-7.53 -4.74,11.67 11.67,0 0,1 -0.37,-4.71c0.77,-7 1.69,-13.95 2.57,-20.92 0.31,-2.49 0.59,-5 1,-7.48a7.5,7.5 0,0 1,2.36 -4.21c5.93,-5.9 11.81,-11.84 17.76,-17.71a2.46,2.46 0,0 0,0.78 -2.62c-2.14,-9.45 -4.2,-18.92 -6.29,-28.38C84,77.33 83.87,77.08 83.69,76.54Z"/>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/8aa95bc22269ef11a970c62e715e84ca8f84ab7e/app/src/main/res/drawable/ic_tutorial_img_2.xml#L3"></span>Very long vector path (1434 characters), which is bad for performance. Considering reducing precision, removing minor details or rasterizing vector.</td>
    </tr>
  </tbody>
</table>